projects/ontimize-web-ngx/src/lib/interfaces/o-table-export-data.interface.ts

```diff
@@ -0,0 +1,37 @@
+import { QueryParameter } from "../types/query-parameter.type";
+
+export interface OTableExportData {
+  data: any;
+  columns: string[];
+  columnNames: object;
+  sqlTypes: { [columnId: string]: string; };
+  filter: object
+}
+
+export interface OTableExportData3X {
+  type: string;
+  queryParam: QueryParameter;
+  service?: string;
+  path: string;
+  dao: string;
+  advQuery: boolean
+}
+
+export interface OTableExcelExportData3X extends OTableExportData3X {
```

Comment:
Change this interface name by a generic one due to it is also used into PDF export operation